helper/labelSchema.js

```diff
@@ -13,12 +13,17 @@ module.exports = {
   'USA': {
     'borough': getFirstProperty(['borough']),
     'local': getFirstProperty(['locality', 'localadmin', 'county']),
-    'regional': getUsOrCaState,
+    'regional': getUsOrCaOrAusState,
     'country': getUSACountryValue
   },
+  'AUS': {
+    'local' : getFirstProperty(['locality', 'localadmin']),
+    'regional' : getUsOrCaOrAusState,
```

Comment:
is there a reason that this function isn't called here? why does it just store the variable name?